tools/cvm-image-rewriter/README.md

```diff
@@ -136,19 +137,22 @@ Before running the tool, please choose the plugins that are needed.You can skip
 For example:
 
     ```
-    touch pre-stage/01-resize-image/NOT_RUN
+    touch plugins/01-resize-image/NOT_RUN
     ```
 
 If the guest image is used for CCNP deployment, it's recommended to run below plugin combination according to different initial guest image type.
```

Comment:
I would reword to "it's recommended to run the below plugin combinations depending on which guest image type is used."